### PR TITLE
Remove `FromPlutusType` / `ToPlutusType`, Improve organisation of conversion modules

### PR DESCRIPTION
**Pull request by errfrom**
Closes: https://github.com/Plutonomicon/cardano-transaction-lib/issues/452

### CODE REVIEW

File: src/Plutus/Conversion.purs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+module Plutus.Conversion
```
Comment (klntsky) [inline src/Plutus/Conversion.purs:1]: Please add some comments about the purpose of this module.

File: package-lock.json [hunk 2]
```diff
@@ -1,5135 +1,13 @@
 {
   "name": "cardano-transaction-lib",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```
Comment (klntsky) [inline package-lock.json:6]: Do you know why this file reduced its size that much?

**Review by klntsky — DISMISSED** [review]
Overall great, thanks!

Please add some docs to the module header

File: package-lock.json (hunk 2 above)
Comment (ngua) [reply package-lock.json:6]: It's a different lockfile version I think

Comment (errfrom) [reply package-lock.json:6]: @klntsky It's a correctly re-generated `package-lock.json`. It looks like I generated it the wrong way before, sorry for that.

**Review by ngua — APPROVED** [review]
Nice work!

**Comment by ngua** [comment]
@errfrom After fixing the merge conflicts, could you also add a line to the changelog under `## [Unreleased]` (using the heading ### Removed)` please?

**Review by ngua — APPROVED** [review]

**Review by ngua — APPROVED** [review]
LGTM

**Pull request merged**